libs/ui-toolkit/src/components/async-renderer/async-renderer.tsx

```diff
@@ -4,29 +4,41 @@ import { t } from '@vegaprotocol/react-helpers';
 
```

Comment:
chore (non-blocking): I have no other changes to be annoying about in this PR so I am going to be annoying in a different way and ask for tests/stories on this file as they are missing and you have changed it. 😂 

You don't really have to but if you have some time it would be good. Else just create a ticket to do it 👍

Reply:
haha ok dude! good plan.